docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/multiple_cluster/index.md

```diff
@@ -42,18 +42,18 @@ It is important to understand that a cell's configuration has the notion of scop
 
 Within this concept of scope, an important point is that all enterprise applications are cell-scoped. In other words, there can be only one enterprise application with a given name in the cell. If multiple servers and clusters, or multiple clusters require the use of that enterprise application, they must share it.
 
-Typically, when you install an enterprise application that is shared across multiple clusters, the administrator installs the enterprise application archive into the cell's Deployment Manager. Then, it maps the application to the target clusters where it runs. HCL Portal installs several enterprise applications as part of its basic configuration and before any cluster is defined. Special steps must be followed to ensure that these infrastructure applications are appropriately shared when multiple clusters are defined within the same cell. And by extension, since they are infrastructure applications, all HCL Portal clusters must be at the same version.
+Typically, when you install an enterprise application that multiple clusters share, the administrator installs the enterprise application archive into the cell's Deployment Manager. Then, it maps the application to the target clusters where it runs. HCL Portal installs several enterprise applications as part of its basic configuration and before any cluster is defined. You must follow special steps to ensure that these infrastructure applications are appropriately shared when multiple clusters are defined in the same cell. And by extension, because they are infrastructure applications, all HCL Portal clusters must be at the same version.
 
-Since portlets are enterprise applications of a special type, it is possible, but not always appropriate, to share portlets across multiple clusters. Many portlets \(for example HCL Portal administration\) are considered part of the infrastructure, and as a result can be shared across multiple clusters. Most user application portlets are specific to certain clusters and are installed as such.
+Because portlets are enterprise applications of a special type, it is possible, but not always appropriate, to share portlets among multiple clusters. Many portlets \(for example HCL Portal administration\) are considered part of the infrastructure, and as a result can be shared across multiple clusters. Most user application portlets are specific to certain clusters and are installed as such.
```

Comment:
```suggestion
Because portlets are enterprise applications of a special type, it is possible, but not always appropriate, to share portlets among multiple clusters. Many portlets (for example, HCL Portal administration) are considered part of the infrastructure, and as a result can be shared across multiple clusters. Most user application portlets are specific to certain clusters and are installed as such.
```